File: RSMK2/RSMK2/Branch.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RSMK2
{
    [Serializable]
    public class Branch
    {
        private string _tree;
        private string _description;
        
        public Branch()
        {
            _tree        = string.Empty;
            _description = string.Empty;
        }

        public Branch(string tree, string description)
        {
            _tree        = tree;
            _description = description;
        }

        public string GetTree() => _tree;
        public void SetTree(string tree) { _tree = tree; }
        public string GetDescription() => _description;
        public void SetDescription(string description) { _description = description; }
    }
}

File: zwsserver/src/types/Branch.java
package types;

public class Branch {

    private String tree;
    private String description;

    public String getTree() {
        return tree;
    }

    public void setTree(String tree) {
        this.tree = tree;
    }

    public Branch() {
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDescription() {
        return description;
    }

    public Branch(String tree, String description) {
        this.tree = tree;
        this.description = description;
    }

    @Override
    public String toString(){
        return "Дерево для веника - " + tree;
    }

}
